federator/integrationTest.js

```diff
@@ -212,13 +218,146 @@ async function transfer(originFederators, destinationFederators, config, origin,
             logger.warn(`Wrong Sender balance. Expected ${expectedBalance} but got ${senderBalanceAfter}`);
         }
 
-        const crossBackCompletedBalance = await originWeb3.eth.getBalance(userAddress);
+        let crossBackCompletedBalance = await originWeb3.eth.getBalance(userAddress);
         logger.debug('Final user balance', crossBackCompletedBalance);
         logger.debug('Cost: ', BigInt(initialUserBalance) - BigInt(crossBackCompletedBalance));
 
+
+        logger.info('------------- START TOKEN FALLBACK TEST -----------------');
```

Comment:
In the logger info besides the name of the function metion the type of contract it is, in this case ERC677